src/main/resources/application.yml

```diff
@@ -23,7 +23,12 @@ s3:
   secure: ${S3_SECURE:true}
   root-ca: ${S3_ROOT_CERT_PATH:/etc/ssl/certs/ca-certificates.crt}
 
-jwt.public-key-path: ${JWT_PUBLIC_KEY_PATH:/etc/ega/jwt/key.pem}
+ga4gh:
+  passport:
+    openid-configuration-url: ${OPENID_CONFIGURATION_URL:https://login.elixir-czech.org/oidc/.well-known/openid-configuration}
+    public-key-path: ${PASSPORT_PUBLIC_KEY_PATH:/etc/ega/jwt/passport.pem}
+  visa:
+    public-key-path: ${VISA_PUBLIC_KEY_PATH:/etc/ega/jwt/visa.pem}
```

Comment:
there might be multiple of visa public keys e.g. one from EGA, one from REMS instance one from Elixir AAI .. how do we handle this ?

Reply:
Hmmm... I assumed that the Visas of interest will be always signed by one facility (either by EGA, in case of FederatedEGA, or by REMS, in case of standalone installation). If it's not the case, I'll have to add support for multiple injected public keys. But can we postpone it? This is anyway not a generic, but TSD case: AFAIK, you are not going to inject those keys, because you will rely on JKU.

Reply:
true.